packages/core/src/lib/ConnectionManager.ts

```diff
@@ -0,0 +1,336 @@
+import { Connection } from "@libp2p/interface-connection";
+import { PeerId } from "@libp2p/interface-peer-id";
+import { Peer } from "@libp2p/interface-peer-store";
+import { IRelay, Tags } from "@waku/interfaces";
+import debug from "debug";
+import { Libp2p } from "libp2p";
+
+import { createEncoder } from "../index.js";
+
+import { RelayPingContentTopic } from "./relay/constants.js";
+
+const log = debug("waku:connection-manager");
+
+const DEFAULT_PEER_DISCOVERY_CONNECTION_INTERVAL = 10 * 1000;
```

Comment:
```suggestion
const DEFAULT_PEER_DISCOVERY_CONNECTION_INTERVAL_MS = 10 * 1000;
```

I think it's always best to include the unit so there is no mistake